python_sdk/infrahub_sdk/uuidt.py

```diff
@@ -39,9 +41,22 @@ def __init__(
         hostname: Optional[str] = None,
         random_chars: Optional[str] = None,
     ):
-        self.namespace = namespace or str(Path(__file__).parent.resolve())
```

Comment:
Are you seeing some performance issue related to the uuidt ?

Reply:
Yes, I'm trying to get rid of the following regression: https://github.com/opsmill/infrahub/pull/3541

Reply:
Same with the gethostname()

Reply:
Indeed... thanks for the tip, I become a python newbie sometimes :)